docs/data/sdks/android-kotlin/index.md

```diff
@@ -53,6 +53,25 @@ val amplitude = Amplitude(
 )
 ```
 
+#### EU data residency
+
+You can configure the server zone when initializing the client for sending data to Amplitude's EU servers. The SDK sends data based on the server zone if it's set.
+
+!!!note
+    For EU data residency, the project must be set up inside Amplitude EU. You must initialize the SDK with the API key from Amplitude EU.
+
+=== "Kotlin"
+
+    ```kotlin
+    val amplitude = Amplitude(
+      Configuration(
+        apiKey = AMPLITUDE_API_KEY,
+        serverZone = ServerZone.US
+      )
+    )
```

Comment:
@qingzhuozhen can you check if the syntax is correct? thank you!

Reply:
looks right! If for Android, maybe also add the context since that is required. There is also another way in Android to init
```
val amplitude = Amplitude(AMPLITUDE_API_KEY, context) {
    this.serverZone = ServerZone.EU
}
```
Either way works now